spec/controllers/my/shares_controller_spec.rb

```diff
@@ -24,6 +24,10 @@
           r.edit_users += [@other_user.user_key]
           r.save!
         end
+        @my_collection = Collection.new(title: "My collection").tap do |c|
+          c.apply_depositor_metadata(@user.user_key)
+        end
+        @my_collection.save!
       end
```

Comment:
@val99erie I know this is unrelated to the fix you added, but we should change this over to the let syntax and clean up the objects after the test completes.  This test is leaving files (and now a collection) behind that could corrupt other tests.